_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -80,13 +80,20 @@ struct CodecKey {
 };
 
 struct Handlers {
+    typedef std::function<VideoENCODE*(VideoCORE* core, mfxCoreInterface* coreInt, mfxStatus *mfxRes)> CtorFunc;
```

Comment:
Better to define such typedefs right away in prev. patches rather than modify something you just recently introduced.

Reply:
Just reuse the whole Funcs structure for fallbacks right away. I don't see the need to create 2 structures. Yes, you will not use some members in fallbacks, but I don't see this as a problem.